src/test/java/automation/tests/LoginPageTest.java

```diff
@@ -32,18 +33,22 @@ void checkIfLoginDefaultUser() {
     }
 
     @BeforeEach
-    public void beforeEach(){
-        driver.get(envConfLoader.getURL());
+    void beforeEach(){
+        go.to(Pages.LOGIN_PAGE);
         loginPage = new LoginPage(driver);
     }
 
     @BeforeAll
-    public static void beforeAll(){
+    static void beforeAll(){
+        EnvironmentConfigLoader envConfLoader = new EnvironmentConfigLoader("environment");
         driver = envConfLoader.getDriver();
+        go = new Go(driver);
+
+        go.to(Pages.LOGIN_PAGE);
```

Comment:
There's no sense to do the same thing (go to login page) in both beforeAll() and beforeEach()

Reply:
Don't know why I do this, but remove one.